playbooks/openshift-etcd/private/migrate.yml

```diff
@@ -19,7 +19,6 @@
       name: etcd
       tasks_from: migrate.pre_check.yml
     vars:
-      r_etcd_common_embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
```

Comment:
It's no longer possible for this to be set to false anywhere.  If oo_etcd_to_config is len 0, evaluate_groups will fail every play.